core/account/builder.go

```diff
@@ -123,7 +123,11 @@ func (a *spendUTXOAction) Build(ctx context.Context, b *txbuilder.TemplateBuilde
 		outid = *a.OutputID
 	} else if a.TxHash != nil && a.TxOut != nil {
 		// This is compatibility layer - legacy apps can spend outputs via the raw <txid:index> pair.
-		outid = bc.ComputeOutputID(*a.TxHash, *a.TxOut)
+		q := `SELECT output_id FROM account_utxos WHERE tx_hash=$1 AND index=$2`
+		err := a.accounts.utxoDB.db.QueryRow(ctx, q, *a.TxHash, *a.TxOut).Scan(&outid)
+		if err != nil {
+			return err
```

Comment:
If the lookup fails to find a matching output, we might want to return `pg.ErrUserInputNotFound` like we do within the reserver: https://github.com/chain/chain/blob/main/core/account/reserve.go#L424

o/w, I think this would end up as a 500?